feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -152,8 +152,13 @@ internal class HomeViewModel @Inject constructor(
                         feedingRouteState = feedingRouteUpdate,
                     )
                 }
-                if (homeStateFlow.locationState !is LocationState.UndefinedLocation) {
-                    nearestFeedingJob.start()
+
+                if (feedingPointUpdate.feedingPoints.isNotEmpty()) {
```

Comment:
Please create a separate method for this block. E. g. `tryToFocusNearestFeedingPoint`